frontend/src/index.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
```

Comment:
Consider adding separate dev and prod `src/index.js` files and base their loading around process.env.NODE_ENV. This way, you don't have to worry about development tooling making it into production, possibly exposing sensitive information.

Reply:
Will address this in a separate commit